src/main/java/gregtech/api/metatileentity/multiblock/RecipeMapMultiblockController.java

```diff
@@ -137,6 +152,7 @@ private void resetTileAbilities() {
         this.outputInventory = new GTItemStackHandler(this, 0);
         this.outputFluidInventory = new FluidTankList(true);
         this.energyContainer = new EnergyContainerList(Lists.newArrayList());
+        this.refreshBeforeConsumptions = new ObjectArrayList<>(0);
```

Comment:
```suggestion
        this.refreshBeforeConsumptions.clear();
```
Requires the change to ArrayList from above.

Reply:
Fixed in new commit.